envs/example/defaults.yml

```diff
@@ -279,45 +293,45 @@ keystone:
     - name: keystone
       type: identity
       description: 'Identity Service'
-      public_url: https://{{ endpoints.main }}:5001/v2.0
-      internal_url: https://{{ endpoints.main }}:5001/v2.0
-      admin_url: https://{{ endpoints.main }}:35358/v2.0
+      public_url: https://{{ endpoints.main }}:5000/v2.0
```

Comment:
I looked into this. Correct me if I'm wrong. I tried to add the port variable in the endpoint, but since it was defined in the same block the ursula deployed failed .